File: 0027-remove-element/0027-remove-element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        count = 0;
        scan = 0;
        while scan < len(nums):
            if nums[scan] != val:
                nums[count] = nums[scan]
                count += 1
            scan += 1
        return count

File: 0027-remove-element/0027-remove-element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int count = 0;
        int scan = 0;
        while(scan<nums.size()){
            if(nums[scan] != val){
                nums[count] = nums[scan];
                count++;
            }
            scan++;
        }
        return count;
        
    }
};